packages/explorer/src/app/(explorer)/layout.tsx

```diff
@@ -17,7 +17,10 @@ const jetbrains = JetBrains_Mono({
 });
 
 export const metadata: Metadata = {
-  title: "World Explorer",
+  title: {
+    default: "World Explorer",
```

Comment:
World Explorer or World**s** Explorer?

Reply:
Yeah good observation, updated to "Worlds"